### PR TITLE
Automatically select 'Download' for store functions

### PR DESCRIPTION
**Pull request by kenfdev**
Signed-off-by: Ken Fukuyama <kenfdev@gmail.com>

## Description

This minor fix selects the 'Download' radio button by default if the function has a label with `com.openfaas.ui.ext`. This will improve the UX for people who use the function store.

## Motivation and Context

- [x] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))
#827 

## How Has This Been Tested?

**Function with `com.openfaas.ui.ext` label**

1. Deploy the QRCode function which has the `com.openfaas.ui.ext` label.
2. Select the function from the list.
3. Check that the `Download` button is selected by default.

**Function without `com.openfaas.ui.ext` label**

1. Deploy the Figlet function which doesn't have the `com.openfaas.ui.ext` label.
2. Select the function from the list.
3. Check that the `Text` button is selected by default (not the Download button).

Tested with:

* Chrome
* Safari
* Firefox
* Edge
* IE11

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by kenfdev** [comment]
default_download

File: gateway/assets/index.html [hunk 1]
```diff
@@ -158,8 +158,8 @@ <h1 style="flex: 1 1 auto;">&nbsp; OpenFaaS Portal</h1>
                                 <md-input-container class="md-block" flex-gt-sm>
                                     <md-radio-group ng-model="invocation.contentType" layout="row" layout-align="start center">
                                         <md-radio-button value="text" class="md-primary"> Text </md-radio-button>
-                                        <md-radio-button value="json"> JSON </md-radio-button>
-                                        <md-radio-button value="binary"> Download </md-radio-button>                                        
+                                        <md-radio-button value="json" class="md-primary"> JSON </md-radio-button>
+                                        <md-radio-button value="binary" class="md-primary"> Download </md-radio-button>
```
Comment (kenfdev) [inline gateway/assets/index.html]: I also fixed the radio button colors of JSON and Download. Somehow they were pink while the Text radio button is Blueish.

@alexellis Do you know if this is intended? If it is, I'll revert it.

Comment (alexellis) [reply gateway/assets/index.html]: I'm not sure.

Comment (kenfdev) [reply gateway/assets/index.html]: **Before**
ss 2018-08-23 22 07 20

**After**
ss 2018-08-23 22 06 30

Comment (kenfdev) [reply gateway/assets/index.html]: I think it's natural to have them all with the same color (blueish, purple)

**Comment by ivanayov** [comment]
Looks good

File: gateway/assets/index.html (hunk 1 above)
Comment (alexellis) [reply gateway/assets/index.html]: Agreed.

**Pull request merged**